src/assets/js/application.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-env browser */
+
+/* global ClipboardJS:false */
+
+import ClipboardJS from 'clipboard';
+
+const btnTitle = 'Copy to clipboard'
+
+const btnHtml = [
+'<div class="bd-clipboard">',
+  '<button type="button" class="btn-clipboard" title="' + btnTitle + '">',
+    '<svg class="bi" role="img" aria-label="Copy"><use xlink:href="#clipboard"/></svg>',
+  '</button>',
+'</div>'].join('');
+
+document.querySelectorAll('div.highlight')
+  .forEach((element) => {
+    element.insertAdjacentHTML('beforebegin', btnHtml)
+  })
+
+window.addEventListener('load', () => {
```

Comment:
Just wondering, shouldn't we use `DOMContentLoaded` instead? My guess is that due to async loading we need to wait, but I'm thinking how we can improve this later.

Reply:
The best optimization is to not block anything unless it's needed. So, in this case `DOMContentLoaded` seems a better choice.

Later, we could look into just building the Bootstrap parts we need and thus we probably won't need this code at all.